lib/core/src/text/buffer/glyph_square.rs

```diff
@@ -225,11 +227,12 @@ impl<'a> GlyphAttributeBuilder for GlyphTextureCoordsBuilder<'a> {
 
     /// Compute texture coordinates for `ch`
```

Comment:
This doc comment is not a sentence + it does not describe all the things it should describe. By inspecting the code I understand that it also takes into consideration previous glphs to get proper spacing right?

Reply:
No, not this function. It only compute texture coordinates, and that depends on single char only